### PR TITLE
Fixed #3654 - Email Quotation

### LINKED ISSUE #3654: Leftover call to Emails/index.php in v7.9, breaks "Email Quotation" feature
#### Issue
When using the "Email Quotation" action from the Quotes detail view, we get a blank screen and nothing happens.

The logs contain this
```
Mon Jun  5 16:39:10 2017 [1252][1][FATAL] emails_email_templates for emails_email_templates failed to load
Mon Jun  5 16:39:10 2017 [1252][1][FATAL] Failed to load relationship emails_email_templates while saving Emails
Mon Jun  5 16:39:10 2017 [1252][1][FATAL] emails_email_templates_idb for emails_email_templates failed to load
```

#### Expected Behavior
Option should work as in previous versions.

#### Possible Fix
It seems there is a leftover called to a file that was removed. The call is in `modules/Emails/compose.php`, line 72:
```
include('modules/Emails/index.php');
``` 
This file seems to have been deprecated and removed in v 7.9. I don't know exactly what should be put in the place of this call, though.

#### Steps to Reproduce
1. Create a Quotes Email Template (in PDF Templates module) if you don't have one yet
2. Create a Quote and Save it
3. From the Detail view of that Quote, select action `Email Quotation` from drop-down action menu

Note that there might be other ways to trigger this bug.

#### Context
This came from [a Forum thread](https://suitecrm.com/forum/suitecrm-7-0-discussion/14486-suite-7-9-0-email-quotation) reported by user `TaufiqueAhmed` and solved by user `xoanon`. 

#### Your Environment
* SuiteCRM Version used: 7.9

### PR DESCRIPTION
**Pull request by daniel-samson**
task: SCRM-604
hotfix: #3654 
## Description

Users are unable to email a quotation with the new compose view.

## Motivation and Context

* Adds the missing functionality to email a quote

## How To Test This

* Repair and Rebuild
* Create a quote
* Action menu -> "Email Quotation"

You should see the compose view with the selected pdf template

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Emails/EmailsController.php [hunk 1]
```diff
@@ -489,6 +504,10 @@ public function composeBean($request, $mode = self::COMPOSE_BEAN_MODE_UNDEFINED)
         if ($mode === self::COMPOSE_BEAN_MODE_REPLY_TO || $mode === self::COMPOSE_BEAN_MODE_REPLY_TO_ALL) {
             // Add Re to subject
             $this->bean->name = $mod_strings['LBL_RE'] . $this->bean->name;
+        } else if($mode === self::COMPOSE_BEAN_WITH_PDF_TEMPLATE) {
+            // get subject from template
+            // Don not add $mod_strings['LBL_NO_SUBJECT']
+            // $this->bean->name = $this->bean->name;
```
Comment (samus-aran) [inline modules/Emails/EmailsController.php]: This else IF does not have any actions?

File: modules/Emails/EmailsController.php [hunk 2]
```diff
@@ -509,6 +528,10 @@ public function composeBean($request, $mode = self::COMPOSE_BEAN_MODE_UNDEFINED)
             }
         }
 
+        if ($mode === self::COMPOSE_BEAN_WITH_PDF_TEMPLATE) {
+
```
Comment (samus-aran) [inline modules/Emails/EmailsController.php]: No Action?

File: modules/Emails/EmailsController.php [hunk 3]
```diff
@@ -475,6 +487,9 @@ public function composeBean($request, $mode = self::COMPOSE_BEAN_MODE_UNDEFINED)
             if ($mode === self::COMPOSE_BEAN_MODE_FORWARD) {
                 $this->bean->to_addrs = '';
                 $this->bean->to_addrs_names = '';
+            } else if($mode === self::COMPOSE_BEAN_WITH_PDF_TEMPLATE) {
+                // Get Related To Field
+                // Populate to
```
Comment (samus-aran) [inline modules/Emails/EmailsController.php]: No Action?

**Comment by daniel-samson** [comment]
Update pull request. When emailing a quotation the subject appears (no subject)<template subject line>

**Comment by daniel-samson** [comment]
This also fixes the issue #3787.

**Pull request merged**